src/action.ts

```diff
@@ -111,22 +148,26 @@ async function linkAllFilesInMetafile(
 
 	if (decision === Decision.MATCH) {
 		return resultOf(linkExactTree(sourceRoot, fullLinkDir));
-	} else {
+	} else if (decision === Decision.MATCH_SIZE_ONLY) {
 		return resultOf(
 			fuzzyLinkOneFile(searchee, newMeta, fullLinkDir, sourceRoot)
 		);
+	} else {
+		return resultOf(
+			fuzzyLinkPartial(searchee, newMeta, fullLinkDir, sourceRoot)
+		);
```

Comment:
could there be any "collision" between these two match types that would mean the ordering of these branches matters?

Reply:
I don't think that's possible. We would want this ordering anyway as it preserves the most information to match with the most restrictive to the least.